tests/Makefile

```diff
@@ -225,9 +225,9 @@ decodecorpus : $(filter-out zstdc_zstd_compress.o, $(ZSTD_OBJECTS)) $(ZDICT_FILE
 symbols  : symbols.c zstd-dll
 ifneq (,$(filter Windows%,$(OS)))
 	cp $(ZSTDDIR)/dll/libzstd.dll .
-	$(CC) $(FLAGS) $< -o $@$(EXT) -DZSTD_DLL_IMPORT=1 libzstd.dll
+	$(CC) $(FLAGS) $< -o $@$(EXT) -fvisibility=hidden -DZSTD_DLL_IMPORT=1 libzstd.dll
 else
-	$(CC) $(FLAGS) $< -o $@$(EXT) -Wl,-rpath=$(ZSTDDIR) $(ZSTDDIR)/libzstd.so   # broken on Mac
+	$(CC) $(FLAGS) $< -o $@$(EXT) -fvisibility=hidden -Wl,-rpath=$(ZSTDDIR) $(ZSTDDIR)/libzstd.so   # broken on Mac
```

Comment:
I think that `-fvisibility=hidden` is a directive that must be passed _at dynamic library compilation time_.
It sets the symbols that can be accessed at link time when a program links to the dynamic library.

Which means that, when compiling the caller program (`symbols` in this case), it's no longer useful to specify `-fvisibility=hidden` : the list of accessible symbols is already set within `libzstd.so` or `libzstd.dll`.

Reply:
Just realized this is already being done over here: 

https://github.com/facebook/zstd/blob/6a426ef65e0864b658dc78bbb45eb23c29b1275d/lib/Makefile#L169

So I guess I don't need this pr at all and can just move forward with trying to deprecate symbols.c by ensuring all symbols are called by unit tests in a following pr?

Reply:
I agree

Reply:
Okay, I'll go ahead and close this PR then.